Description: Fake that Apache is installed.
 Instead of checking version number with apache2 -v, we hard-code a working
 version. This enables the package to be built without installing a running
 Apache on the building host.
Author: Steinar H. Gunderson <sesse@debian.org>
Origin: vendor
Forwarded: not-needed
Last-Update: 2010-02-28

--- libapreq2-2.12.orig/build/version_check.pl
+++ libapreq2-2.12/build/version_check.pl
@@ -6,7 +6,7 @@ GetOptions(\my %opts, "version=s");
 my ($tool, $path) = @ARGV;
 $path = $tool unless defined $path;
 
-sub exe_version { scalar qx/$path -v/ }
+sub exe_version { return "2.0.53"; }
 sub gnu_version { scalar qx/$path --version/ }
 
 sub xsb_version {
